fileServer.js

```diff
@@ -0,0 +1,90 @@
+ 'use strict';
+
+ var http = require('http');
+ var fs = require('fs');
+ var path = require('path');
+ var url = require('url');
+ var rootDir = __dirname + '/public';
+
+ const mimeType = {
+   'html': 'text/html',
+   'jpeg': 'image/jpeg',
+   'jpg': 'image/jpeg',
+   'png': 'image/png',
+   'js': 'text/javascript',
+   'css': 'text/css'
+ };
+
+ function generate500error(res, errorMessage) {
+   if (errorMessage === 'Internal Server Error') {
```

Comment:
this logic is flawed :). What if the error message does not equal 'Internal Server Error' ?